Review.md

```diff
@@ -1,6 +1,13 @@
 # Please answer the following questions
 
 1.  In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+- Describe is a way to label a group of tests based on the larger concept they are testing. it() is another way to write test() and runs a specific test.
+
 2.  What is the point of `Test Driven Development`? What do you think about this approach?
-3.  What are `mocks`? What are a good use cases for them?
-4.  Mention three types of automated tests.
+
```

Comment:
Looks like you left off mocks! I would recommend reading through this article https://jestjs.io/docs/en/mock-functions